test/testprocess.c

```diff
@@ -530,13 +799,30 @@ static const SDLTest_TestCaseReference processTestMultiprocessStdinToStdout = {
     process_testMultiprocessStdinToStdout, "process_testMultiprocessStdinToStdout", "Test writing to stdin and reading from stdout using the simplified API", TEST_ENABLED
 };
 
+static const SDLTest_TestCaseReference processTestWriteToFinishedProcess = {
+    process_testWriteToFinishedProcess, "process_testWriteToFinishedProcess", "Test writing to stdin of terminated process", TEST_DISABLED
```

Comment:
```suggestion
    process_testWriteToFinishedProcess, "process_testWriteToFinishedProcess", "Test writing to stdin of terminated process", TEST_ENABLED
```
I disabled the test for ci to run the other tests.